src/language-map.ts

```diff
@@ -43,4 +47,9 @@ export const languageMap: Record<SupportedLanguage, IPrettierOption> = {
     parser: 'css',
     plugins: [parserCSS]
   },
+  ,
```

Comment:
I think this should be removed.
```suggestion
```

Reply:
Sorry, maybe the Easter chocolate effect. This fixes the problem: commit 25192d2